### DISCUSSION TITLE
Hotkey support (new chat for example)

### DISCUSSION THREAD

**Discussion by illgitthat** [danny-avila/LibreChat#2645] [Feature Requests & Suggestions]
I'm not sure what the best hotkeys are for web applications (that won't conflict with system level or browser level), but these would be nice to have.

Some ideas where hotkeys would be nice:
* New chat
* Something like on discord, hold shift when hovering the convo pre-open the menu, and deleting the convo while holding shift bypass the popup

Some strategies for this:
* Single letter hotkeys are common in some web applications (Gmail/Outlook, Facebook) which of course would not apply when text edit box is active
* Otherwise.... ctrl + m for new chat perhaps?

**Comment by danyo1399** [comment]
chatgpt has keyboard shortcuts already and they work well. copy paste?

**Comment by mkagit** [comment]
ESC to move the cursor to the search input

**Comment by danielrosehill** [comment]
Great suggestion!

The ones that I would like (Linux desktop, frequently copying outputs containing code snippets into a notepad editor):

-> Start voice dictation
-> End voice dictation
-> New chat

Additional hotkeys that would be very helpful as "great to haves"

-> Forking outputs
-> Copy last completion (to clipboard)
-> Copy all completions / whole chat (to clipboard)
-> Copy only prompt (to clipboard)

**Comment by barruda** [comment]
followup: 
https://github.com/danny-avila/LibreChat/issues/3664

**Comment by leikoilja** [comment]
Hey guys, just popping in to say that in a meantime, before we get a proper native support for keyboard shortcuts in librechat, I have added basic keyboard shortcuts to the desktop "wrapper" of the app (you can find it [here](https://github.com/leikoilja/LibreChat-UI/releases/tag/v1.3.0)).
Simple things that i use daily like "create a new chat" and "archive the current" chat are now available from the keyboard 🥳
